solr/bin/solr

```diff
@@ -1185,6 +1185,8 @@ if [ "${SOLR_MODE:-}" == 'solrcloud' ]; then
   : "${ZK_CLIENT_TIMEOUT:=30000}"
   CLOUD_MODE_OPTS=("-DzkClientTimeout=$ZK_CLIENT_TIMEOUT")
 
+  echo -e "\nSolr will start in SolrCloud mode by default in version 10, and you will no longer need to pass in -c or --cloud flag.\n"
```

Comment:
For the record, you can also start solr in cloud mode with `--zk-host` or `-DzkRun` arguments, or by setting `ZK_HOST` env.var. So perhaps the warning should be:
```suggestion
  echo -e "\nSolr will start in SolrCloud mode by default in version 10, even if no -c or --cloud flag is specified.\n"
```